.nvmrc

```diff
@@ -1 +1 @@
-14
+18.16
```

Comment:
imo, I believe it would be beneficial to update Node.js to the latest version (18.16) with its corresponding npm version, since version 16 is no longer going to be supported. 
References:
https://github.com/nodejs/release#release-schedule
https://nodejs.org/en/download

Reply:
we might actually want to leave this slightly more vague by specifying `18` as opposed to locking a specific version of 18